flocker/volume/filesystems/zfs.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import absolute_import
 
 import os
+import libzfs_core
```

Comment:
Publishing libzfs_core on PyPI is the plan.
I think that we should do that after its review is completed.